src/App.tsx

```diff
@@ -0,0 +1,139 @@
+
+    return valorFormatado
+  }
+
+  return (
+    <>
+      <main className='container'>
+        <img 
+          className='logoImg'
+          src={logoImg} 
+          alt='Logotipo de um carro no posto de combustível com um relógio em cima escrito TRAVEL CALCULATOR' 
+        />
+        <form className='form' onSubmit={calcular}>
+          <label>
+            Distância (km):
+            <input 
+              className='input' 
+              placeholder='100' 
+              type='number' 
+              min={0} 
+              step={0.01} 
+              required 
+              value={distancia}
+              onChange={ (e) => setDistancia(Number(e.target.value)) }
+            />
+          </label>
+          <label>
+            Consumo médio do veículo (km/l):
+            <input 
+              className='input' 
+              placeholder='40' 
+              type='number' 
+              min={0} 
+              step={0.01}
+              required 
+              value={consumo} 
+              onChange={ (e) => setConsumo(Number(e.target.value)) }
+            />
+          </label>
+          <label>
+            Preço do combustível (R$):
+            <input 
+              className='input' 
+              placeholder='6,20' 
+              type='number' 
+              min={0} 
+              step={0.01} 
+              required 
+              value={combustivel} 
+              onChange={ (e) => setCombustivel(Number(e.target.value)) }
+            />
+          </label>
+          <label>
+            Quantidade de pedágios:
+            <input 
+              className='input' 
+              placeholder='2' 
+              type='number' 
+              min={0} 
+              step={1} 
+              required 
+              value={pedagio}
+              onChange={ (e) => setPedagio(Number(e.target.value)) }
+            />
+          </label>
+          {pedagio > 0 && (
+            <label>
+              Valor de cada pedágio (R$):
+              <input 
+                className='input' 
+                placeholder='1,50' 
+                type='number' 
+                min={0} 
+                step={0.01} 
+                required 
+                value={pedagioValor} 
+                onChange={ (e) => setPedagioValor(Number(e.target.value)) }
+              />
+            </label>
+          )}
+          <button type='submit' className='buttonSubmit'>
+            Calcular custo total
+          </button>
+        </form>
+        {info && Object.keys(info).length > 0 && (
+          <section className='cardResultado'>
+            <h2>Resumo da Viagem</h2>
+            <span>Custo com combustível: {formatarMoeda(info.combustivel)}</span>
+            <span>Custo com pedágios: {formatarMoeda(info.pedagio)}</span>
+            <span>Custo total: {formatarMoeda(info.total)}</span>
+            <img className='citroen' src={carro} alt="Carro da citroen cinza me movimento 8-bit" />
+          </section>
```

Comment:
Como o app é simples, está ok não ter componentes, mas se fossemos escalar, seria interessante criar componentes de tudo aqui, como essa section por exemplo.

```suggestion
          <InformationResultCard info={info} />
```

Reply:
Okay, próximo projeto vou atualizar para componentes, é por que o foco desse projeto era mais para praticar o useState e o interface.